developer-docs-site/docs/concepts/coin-and-token/aptos-token-overview.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Aptos Token Standard Overview"
+id: "aptos-token-overview"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Aptos Token Standard Overview
+
+:::tip Aptos token standard
+Also see the official token standard [Aptos Token](/concepts/coin-and-token/aptos-token.md).
+:::
+
+## Introduction
+One of the major application of blockchain is token, which requires a standard defining
```

Comment:
nit, let's try to put this all on a single line, we don't gain much except headaches by trying to have line character limits :/